# Request
std_msgs/Bool carry
std_msgs/String type # carry trap and object differently

---
# Response
std_msgs/Bool success
